src/metpy/calc/tools.py

```diff
@@ -825,6 +825,50 @@ def lat_lon_grid_deltas(longitude, latitude, y_dim=-2, x_dim=-1, **kwargs):
     return dx * units.meter, dy * units.meter
 
 
+@exporter.export
+@preprocess_xarray
+def azimuth_range_to_lat_lon(azimuths, ranges, center_lon, center_lat, **kwargs):
+    """Convert azimuth and range locations in a polar coordinate system to lat/lon coordinates.
+
+    Pole refers to the origin of the coordinate system.
+
+    Parameters
+    ----------
+    azimuths : array_like
+        array of azimuths defining the grid. If not a `pint.Quantity`,
+        assumed to be in degrees.
+    ranges : array_like
+        array of range distances from the pole. Typically in meters.
+    center_lat : float
+        The latitude of the pole in decimal degrees
+    center_lon : float
+        The longitude of the pole in decimal degrees
+    kwargs : arbitrary keyword arguments to pass to pyproj.Geod (e.g. 'ellps')
+
+    Returns
+    -------
+    lon, lat: 2D arrays of longitudes and latitudes corresponding to original locations
+
+    Notes
+    _____
```

Comment:
The fact that this looks like a continuous line on GitHub makes me think some unicode chars slipped in here that's confusing the tools?